include/nvexec/stream/bulk.cuh

```diff
@@ -25,6 +25,7 @@ namespace nvexec::STDEXEC_STREAM_DETAIL_NS {
   namespace _bulk {
     template <int BlockThreads, class... As, std::integral Shape, class Fun>
     __launch_bounds__(BlockThreads) __global__ void kernel(Shape shape, Fun fn, As... as) {
+      static_assert(trivially_copyable<Shape, Fun, As...>);
```

Comment:
Should there be a symmetrical assert in the multi gpu bulk kernel?

Reply:
I'd like to put that assert in every kernel.